test/bin-post2.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
It's not required, but you can use the same workdir for both post-bin.json and post-bin2.json if you want to make it a bit DRYer. All you have to do is set the '--workdir' option in post-bin2.json (or both of them).